libs/generic-view/store/src/lib/backup/reducer.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createReducer } from "@reduxjs/toolkit"
+import { setBackupFiles } from "./actions"
+
+export interface Backup {
```

Comment:
🟢 
Not sure if we want to keep Backup interface in store lib. Maybe it could be moved to model lib.